src/components/Result/Result.styles.ts

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+import { pallete } from "../../misc/pallete";
+import { Results } from "./Result.const";
+
+export const Wrapper = styled.p<{
```

Comment:
tutaj podobnie jak w innym miejscu - Wrapper to raczej nazwa dla pudełek, tutaj mamy jakiś tekst